goverge/test/test_main.py

```diff
@@ -210,6 +218,23 @@ def test_race(self):
             'short': False,
             'tag': None,
             'test_path': None,
+            'threads': 4,
+            'xml': False,
+            'xml_dir': 'xml_reports/'
+        }
+        self.assertEquals(expected, vars(args))
+
+    def test_threads(self):
+        args = main._parse_args(["--threads=10"])
+        expected = {
+            'godep': False,
+            'html': False,
+            'project_import': None,
+            "race": False,
+            'short': False,
+            'tag': None,
+            'test_path': None,
+            'threads': '10',
```

Comment:
You could probably make this more streamlined by having the test be more similar to:

    def test_threads(self):
         args = main._parse_args(["--threads=10"])
         self.assertEquals('10', args.threads)

with a single "default args" test to capture all the defaults.This helps avoid having to change each test case whenever you add/modify a default arg.